modules/update-cvo-conditions.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-update-service.adoc
+
+[id="update-cvo-conditions_{context}"]
+= Understanding CVO conditions
+
+The `ClusterVersion` object sets the conditions that describe the state of the cluster-version operator (CVO).
+
+Failing:: When `Failing` is set to True, the CVO fails to reconcile the cluster with the desired release image. In all cases, the impact on the cluster is that the dependent nodes in the manifest graph may not be reconciled. The graph may be flattened, in which case there are no dependent nodes.
```

Comment:
For; "the CVO fails to reconcile", can we [stick with][1] "is failing to reconcile"?  Because one common misconception is that there is a point where folks can say "this update has failed".  In reality, the CVO is constantly attempting to reconcile the cluster with the `status.desired` release image, and reporting `Failing=True` is "my reconciliation attempts are not going well, but I'm still trying".

Separately, bringing in the "graph may be flattened" language, without [the associated manifest-graph discussion][2] might be confusing.  Maybe these status docs should idle until we have docs which cover that sort of "how updates are currently structured" portion?

[1]: https://github.com/openshift/cluster-version-operator/blob/c07975c452303469c378bc5b0f70d8763eaca1f3/docs/user/status.md#failing
[2]: https://github.com/openshift/cluster-version-operator/blob/c07975c452303469c378bc5b0f70d8763eaca1f3/docs/user/reconciliation.md#manifest-graph